CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to the Wazuh app project will be documented in this file.
 
+## Wazuh v4.4.1 - OpenSearch Dashboards 2.6.0 - Revision 07
+
+### Change
+
+- Change the search method in the agents inventory tables [#5196](https://github.com/wazuh/wazuh-kibana-app/pull/5196)
```

Comment:
Move to `Fixed`

Reply:
Fixed here: https://github.com/wazuh/wazuh-kibana-app/pull/5196/commits/4f8d376e050ab15ed5d9a80ee2b964337f6e7a07